scipy/stats/stats.py

```diff
@@ -1594,16 +1633,29 @@ def kurtosistest(a, axis=0, nan_policy='propagate'):
         Defines how to handle when input contains nan.
         The following options are available (default is 'propagate'):
 
-          * 'propagate': returns nan
-          * 'raise': throws an error
-          * 'omit': performs the calculations ignoring nan values
+        * 'propagate': returns nan
+        * 'raise': throws an error
+        * 'omit': performs the calculations ignoring nan values
+
+    alternative : {'two-sided', 'less', 'greater'}, optional
+        Defines the alternative hypothesis.
+        The following options are available (default is 'two-sided'):
+
+        * 'two-sided': the kurtosis of the distribution underlying the sample
+          is different from that of the normal distribution (i.e. 0)
```

Comment:
At the top, it says:

![image](https://user-images.githubusercontent.com/6570539/111274580-664f8780-85f2-11eb-81a6-2d4c8b452378.png)

I don't understand. What does `3*(n+1)/(n-1)` refer to? 

_Assuming_ `n` is the number of observations in the sample `a` (because it is not stated): 
* the kurtosis of a _population_ does not depend on the number of observations of a sample drawn from it
* what is this formula? We use Fisher's definition of kurtosis, so the kurtosis of the normal distribution is 0, right?

Should I just remove this formula?

Reply:
> the kurtosis of a population does not depend on the number of observations in a sample drawn from it, right?

Right.

> We use Fisher's definition of kurtosis, so the kurtosis of the normal distribution is 0, right?

Right.

> Should I just remove this formula?

Sure, it sounds like the right thing to do!

Reply:
OK, I see from the reference that this is the mean of the distribution of the sample kurtosis (not the excess/Fisher kurtosis) when the sample is drawn from the normal distribution. Using git blame, I see that the original docstring was:
> Tests whether a dataset has normal kurtosis (i.e., kurtosis=3(n-1)/(n+1)).

This makes some sense, as it's a statement about the dataset, and perhaps SciPy's convention of using excess kurtosis was not as fully-formed then.

But given the rest of current docstring, I think it should be removed.

Reply:
I agree. Thanks for noticing this. I am out right now so will push this change when I am back home. I guess this is ready to merge afterwards (unless someone else reviews this), right?

Reply:
I went ahead. If it looks good to you, I'll merge.

Reply:
I also removed "(i.e. 0)" to avoid having to explain the different conventions for kurtosis.

Reply:
Everything looks good to me. So +1 for merging. (My comment about being explicit about the alternate hypothesis is non blocking. So, please don't let it bother you from merging. Seems like I have got something wrong there... )